test/unit/helpers/xhr/FACTOR_ENROLL_ACTIVATE_push_sms.js

```diff
@@ -0,0 +1,83 @@
+define({
```

Comment:
this file is exactly the same as `test/unit/helpers/xhr/FACTOR_ENROLL_ACTIVATE_push_email.js`. Can we have just one file added instead of 2? We can name it just `FACTOR_ENROLL_ACTIVATE_push`

Reply:
Yup